modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -0,0 +1,124 @@
+= Configure Role-based Access Control
+:description: Role-based access controls provide an extension to ACLs for managing permissions at scale.
+:page-categories: Management, Security
+
+Role-based Access Control (RBAC) provides a way to configure permissions for provisioned users at scale. RBAC allows you to manage access for many users to many resources with a streamlined interface. This mechanism works in conjunction with all supported authentication methods.
+
+IMPORTANT: At present, Redpanda only supports RBAC for self-hosted clusters. Configuration is only supported through the command line using RPK.
+
+== RBAC
+
+RBAC is designed to address the critical challenge of access management at scale. It alleviates the process of manually maintaining and verifying a set of raw permissions (ACLs) for a user base that may contain thousands of users. RBAC provides you with a method to onboard new employees easier, audit accesses faster, and adapt more readily to evolutions of usage and compliance needs.
+
+Using RBAC, you can define roles to reflect organizational structure or job duties to implement a coherent approach to access management. This approach decouples users and permissions through the assignment of roles. This allows a one to many mapping of a given role to many users, thereby dramatically reducing the number of custom policies needed for a resource from one per user to one per group of users.
+
+image::shared:rbac-overview.png[RBAC multiplicity]
+
+=== RBAC terminology
+
+Under the RBAC framework, you create *roles*, grant *permissions* to those roles, and assign the roles to *users*. When you change the permissions for a given role, all principals with that role automatically gain the modified permissions. You grant or deny permissions for a role by creating an ACL and specifying the RBAC role as the allowed or denied principal respectively.
+
+==== Roles
+
+A role is a named collection of ACLs which can be assigned to zero or more users. Redpanda provides two default roles, `user` and `admin`:
```

Comment:
```suggestion
A role is a named collection of ACLs that can be assigned to zero or more users. Redpanda provides two default roles, `user` and `admin`:
```

Reply:
How/when would one assign a role to zero users?

Reply:
You wouldn't, but there might exist a role which is not currently assigned to any user principals. An "empty" role is perfectly legal.

Reply:
I'll rephrase -- instead of "can be assigned to zero or more users", "that has zero or more assigned users". I'll also rephrase the bit about two default roles in light of the removal of 'Admins' and the fact that 'Users' is only there for upgraded clusters, not fresh clusters.